docs/digger/digger-and-rhmap.asciidoc

```diff
@@ -1,127 +0,0 @@
----
```

Comment:
this should not be here at all - keep digger and rhmap separated

Reply:
agreed this file should be removed

Reply:
resolved

Reply:
https://github.com/aerogear/aerogear.org/pull/691